app/api/models/domain/DownstreamTaxYear.scala

```diff
@@ -24,11 +25,30 @@ import java.time.format.DateTimeFormatter
   *
   * @param value the tax year string (where 2018 represents 2017-18)
   */
-case class DownstreamTaxYear(value: String) extends AnyVal {
+final case class DownstreamTaxYear private(value: String) {
```

Comment:
needs to be renamed to TaxYear (along with the test). Make sure it's copied from individuals-income-received-api, and the TYS tests are also carried over from TaxYearSpec

Reply:
Fixed